tests/ert_tests/shared/test_main_entry.py

```diff
@@ -120,3 +123,103 @@ def test_vis_database_url_forwarded(monkeypatch):
     mocked_connect_or_start_server.assert_called_once_with(
         res_config=None, database_url="TEST_DATABASE_URL"
     )
+
+
+@pytest.mark.parametrize(
+    "config_content, expected",
+    [
+        pytest.param("--Comment", "", id="Line comment"),
+        pytest.param(" --Comment", "", id="Line comment with whitespace"),
+        pytest.param("\t--Comment", "", id="Line comment with whitespace"),
+        pytest.param("KEY VALUE", "KEY VALUE\n", id="Config line"),
+        pytest.param("KEY VALUE --Comment", "KEY VALUE\n", id="Inline comment"),
+        pytest.param(
+            'FORWARD_MODEL("--argument_or_comment")',
+            'FORWARD_MODEL("--argument_or_comment")\n',
+            id="Not able to determine inline comment or part of config",
+        ),
+    ],
+)
+def test_logging_config(caplog, config_content, expected):
+    base_content = "Content of the configuration file (file_name):\n{}\n"
+    logger = logging.getLogger(__name__)
+    config_path = "file_name"
+    with patch("builtins.open", mock_open(read_data=config_content)), patch(
+        "os.path.isfile", MagicMock(return_value=True)
+    ):
+        with caplog.at_level(logging.INFO):
+            log_config(config_path, logger)
+    expected = base_content.format(expected)
+    result = caplog.text[-len(expected) :]
```

Comment:
Maybe use `caplog.messages` instead? Adornments is probably not of consequence here, we care only about the contents.

Reply:
Forgot about `message`, that makes a lot of sense, updated :+1: